build.gradle

```diff
@@ -22,7 +22,7 @@ ext {
     compileSdkVersion = 30
     minSdkVersion = 21
     targetSdkVersion = 30
-    versionCode = 11
+    versionCode = 12
```

Comment:
I'm curious - what is this?

Reply:
These are supposed to be unique so after we released `3.x` we bumped the verison code here just in case. Although this may apply to actual apps released on the PlayStore and not to SDKs.

Reply:
Ended up bumping the version code out of habit. Similar to the build number on iOS I believe.

Reply:
Awesome, thank you